build.sbt

```diff
@@ -1,57 +1,129 @@
-sbtPlugin := true
-
 name := "sbt-typelevel"
 
-organization := "org.typelevel"
-
-version := "0.4-SNAPSHOT"
-
-licenses := Seq("Apache 2" → url("https://www.apache.org/licenses/LICENSE-2.0.html"))
-
-homepage := Some(url("http://typelevel.org/"))
-
-resolvers += Resolver.sonatypeRepo("releases")
-
-// This is both a plugin and a meta-plugin
-
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.2")
+ThisBuild / tlBaseVersion := "0.4"
+// ThisBuild / organization := "org.typelevel"
+ThisBuild / organization := "com.armanbilge"
```

Comment:
Is this a rights thing or a this-is-still-experimental thing?

Reply:
Rights.

Reply:
If you post on the Sonatype JIRA, I'll vouch for you.

Reply:
@rossabaker https://issues.sonatype.org/browse/OSSRH-76765

Also, would be great to be added to this repo as well. Thanks!

Reply:
Done.  I created an @typelevel/sbt-typelevel team and added @djspiewak and @ChristopherDavenport as well for prior build art.

Reply:
I'm on the team, but don't think I got any elevated permissions.

Reply:
Rights are fixed.  (See #13).